docs/source/en/_toctree.yml

```diff
@@ -18,6 +18,8 @@
       title: Configuring Pipelines, Models, and Schedulers
     - local: using-diffusers/custom_pipeline_overview
       title: Loading and Adding Custom Pipelines
+    - local: using-diffusers/kerascv
```

Comment:
Unsure about this placement. On the one hand, it's related to "loading" weights, but on the other hand is a special purpose thing that requires a previous training.